package.json

```diff
@@ -1,29 +1,43 @@
 {
-	"private": true,
-	"scripts": {
-		"lint": "standard 'packages/**/src/**/*.js' --fix",
-		"bootstrap": "lerna bootstrap",
-		"clean": "gulp clean && lerna clean --yes",
-		"build": "gulp build",
-		"watch": "gulp watch",
-		"publish": "npm run clean && npm run lint && npm run build && lerna publish"
-	},
-	"devDependencies": {
-		"babel-cli": "^6.26.0",
-		"babel-plugin-transform-async-to-generator": "^6.24.1",
-		"babel-plugin-transform-object-rest-spread": "^6.26.0",
-		"babel-plugin-transform-react-jsx": "^6.24.1",
-		"babel-plugin-transform-runtime": "^6.23.0",
-		"babel-preset-env": "^1.6.0",
-		"del": "^3.0.0",
-		"gulp": "^3.9.1",
-		"gulp-babel": "^7.0.0",
-		"gulp-newer": "^1.3.0",
-		"gulp-plumber": "^1.1.0",
-		"gulp-util": "^3.0.8",
-		"husky": "^0.14.3",
-		"lerna": "^2.4.0",
-		"standard": "^10.0.3",
-		"through2": "^2.0.3"
-	}
+  "private": true,
+  "scripts": {
+    "lint": "standard 'packages/**/src/**/*.js' 'test/**/*.js' --fix",
+    "test": "npm run build && npm run lint && jest ./test/*.spec.js",
+    "bootstrap": "lerna bootstrap",
+    "clean": "gulp clean && lerna clean --yes",
+    "build": "gulp build",
+    "watch": "gulp watch",
+    "publish": "npm run clean && npm test && lerna publish"
```

Comment:
I removed the build step here because that happens in `test`, but we do lose some expressiveness in what happens during a publish. If we want that expressiveness back, we could have a `buildAndTest` script which is then called by `npm test` and `npm run publish`.

Reply:
I'm good with this as is.